KenBurns/Classes/RingBuffer.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  RingBuffer.swift
+//  phonewallpapers
+//
+//  Created by Justin Barnett on 2/14/20.
+//  Copyright © 2020 Justin Barnett. All rights reserved.
```

Comment:
Please remove this entire header comment. Any code contributed to this repo is subject to the repo's license.

Reply:
👍🏼Done